db/migrate/20210202145557_create_dealer.rb

```diff
@@ -5,8 +5,6 @@ def change
       t.boolean :fully_staffed
       t.integer :monthly_gross
       t.string :city
-      t.datetime :created_at
```

Comment:
Taking these out doesn't change the columns in our tables after running a new db:migrate?